contracts/interfaces/IPoolCommitter.sol

```diff
@@ -11,6 +11,19 @@ interface IPoolCommitter {
         LongBurn
     }
 
+    // User aggregate balance
+    struct Balance {
+        uint256 longTokens;
+        uint256 shortTokens;
+        uint256 settlementTokens;
+    }
+
+    // Token Prices
+    struct Prices {
+        bytes16 longPrice;
+        bytes16 shortPrice;
```

Comment:
Why `bytes16` here?

Reply:
It saves from having to go from `bytes16 -> uint256 -> bytes16` when you want to use the `price` to calculate using the `ABDKMathQuad` library.

Reply:
Very fair